surveys/2023-annual-survey/questions.md

```diff
@@ -94,6 +94,19 @@ Type: select one
 > those who program once a week but always in Rust and those who program daily but
 > use Rust once a week.
 
+### On average, how often do you do coding/design work in general (i.e. excluding Rust)?
```

Comment:
Design work sounds a bit like graphical design :laughing: But I'm not sure how to formulate it better.

I think that we should just ask about programming/coding, to have better sync with the previous question (do we do software design *in Rust*? hard to say).
Maybe just "how often do you code in any programming language?"